deployment/build.windows.amd64

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+#= Windows 7+ amd64 (x64) .zip
```

Comment:
why no more x86 Windows build?

Reply:
It's a massive hassle and it's 2020 32-bit Windows should be dead.